### DISCUSSION TITLE
Don't dismiss PR approvals if author hasn't changed files owned by approver?

### DISCUSSION THREAD

**Discussion by mariekoho** [community/community#38808] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Would it be possible to add functionality where a PR approval doesn't get dismissed upon PR changes, if the changes don't affect files owned by the approver?

For example, the author tags my team and another team for review on their PR. I look at the changes made to my team's code and approve it on behalf of my team. Someone on the other team requests changes on code owned by their team. The author pushes changes that don't touch code owned by my team, so my approval should remain valid and not be dismissed.

Thank you!

**Comment by ossanna16** [comment]
Hi there @mariekoho and welcome to our community! Thank you for the feedback!

**Comment by blghns** [comment]
Lol I asked the same question here https://github.com/orgs/community/discussions/38806 just before your question
